llvm/include/llvm/IR/Type.h

```diff
@@ -336,9 +354,12 @@ class Type {
 
   /// Return the width of the mantissa of this type. This is only valid on
   /// floating-point types. If the FP type does not have a stable mantissa (e.g.
-  /// ppc long double), this method returns -1.
+  /// ppc long double), or if the type is a decimal floating point this method
+  /// returns -1.
```

Comment:
If we're going with asserting in this method on decimal FP types, this documentation change is incorrect.